src/backend/api/meals.js

```diff
@@ -1,16 +1,165 @@
-const express = require("express");
+const express = require("express")
 const router = express.Router();
 const knex = require("../database");
 
+
+
+
+
+//  /api/meals/:meal_id/reviews	GET	Returns all reviews for a specific meal.
+//  Reviews
+
+router.get("/:meal_id/reviews", async (request, response) => {
+
+
+  let mealReviews = await knex("Review").select().where({ "Review.meal_id": request.params.meal_id })
+
+
+  if (mealReviews.length > 0) {
+    response.json(mealReviews)
+  } else {
+    response.status(404).json({ error: "Review Not found" })
```

Comment:
I wouldn't return a 404 here but instead an empty list (of reviews).

Reply:
Do you mean like this   response.json(mealReviews)? if so why do I need an if statement then?

Reply:
Exactly, you don't need the if statement.

Reply:
okay, thanks.